File: WebApp/Models/AA/AA0027D.cs
using System;
using System.Collections.Generic;

namespace WebApp.Models
{
    public class AA0027D : JCLib.Mvc.BaseModel
    {
        public string MMCODE { get; set; } //院內碼(畫面)
        public string MMCODE_TEXT { get; set; } //院內碼(紅底顯示)
        public string MMNAME_C { get; set; } //中文品名(畫面)
        public string MMNAME_E { get; set; } //英文品名(畫面)
        public string AGEN_NAME { get; set; } //廠商名稱(畫面)
        public string LOT_NO { get; set; } //批號(畫面)
        public string LOT_NO_TEXT { get; set; } //批號(紅底顯示)
        public string EXP_DATE { get; set; } //效期(畫面)
        public string APVQTY { get; set; } //退貨量(畫面)
        public string APVQTY_TEXT { get; set; } //退貨量(紅底顯示)
        public string INV_QTY { get; set; } //效期數量(畫面)
        public string BASE_UNIT { get; set; } //單位(紅底)
        public string C_TYPE { get; set; } //進貨/合約
        public string C_TYPE_NAME { get; set; } //進貨/合約(畫面)
        public string C_TYPE_NAME_TEXT { get; set; } //進貨/合約(紅底)
        public string IN_PRICE { get; set; } //進貨單價(畫面)
        public string CONTPRICE { get; set; } //合約單價(畫面)
        public string C_AMT { get; set; } //換貨金額(畫面)
        public string ITEM_NOTE { get; set; } //備註(畫面)
        public string SEQ { get; set; } //單據項次
        public string UPDATE_USER { get; set; }
        public string UPDATE_IP { get; set; }
        public string UPDATE_TIME { get; set; }
        public string DOCNO { get; set; } //調帳單號
        public string C_UP { get; set; } //退換貨單價
        public string FRWH { get; set; }
    }
}

File: WebApp/Scripts/app/model/AA/AA0027D.js
Ext.define('WEBAPP.model.AA.AA0027D', {
    extend: 'Ext.data.Model',
    fields: [
        { name: 'MMCODE', type: 'string' }, //院內碼(畫面)
        { name: 'MMCODE_TEXT', type: 'string' }, //院內碼(紅底顯示)(畫面)
        { name: 'MMNAME_C', type: 'string' }, //中文品名(畫面)
        { name: 'MMNAME_E', type: 'string' }, //英文品名(畫面)
        { name: 'AGEN_NAME', type: 'string' }, //廠商名稱(畫面)
        { name: 'LOT_NO', type: 'string' }, //批號(畫面)
        { name: 'LOT_NO_TEXT', type: 'string' }, //批號(紅底顯示)
        { name: 'EXP_DATE', type: 'string' }, //效期(畫面)
        { name: 'APVQTY', type: 'string' }, //效期數量(畫面)
        { name: 'APVQTY_TEXT', type: 'string' }, //效期數量(紅底顯示)
        { name: 'INV_QTY', type: 'string' }, //效期數量(畫面)
        { name: 'BASE_UNIT', type: 'string' }, //單位(紅底)
        { name: 'C_TYPE', type: 'string' }, //進貨/合約
        { name: 'C_TYPE_NAME', type: 'string' }, //進貨/合約(畫面)
        { name: 'C_TYPE_NAME_TEXT', type: 'string' }, //進貨/合約(紅底)
        { name: 'IN_PRICE', type: 'string' }, //進貨單價(畫面)
        { name: 'CONTPRICE', type: 'string' }, //合約單價(畫面)
        { name: 'C_AMT', type: 'string' }, //換貨金額(畫面)
        { name: 'ITEM_NOTE', type: 'string' }, //備註(畫面)
        { name: 'SEQ', type: 'string' }, //單據項次
        { name: 'UPDATE_USER', type: 'string' },
        { name: 'UPDATE_IP', type: 'string' },
        { name: 'UPDATE_TIME', type: 'string' },
        { name: 'DOCNO', type: 'string' }, //調帳單號
        { name: 'C_UP', type: 'string' }, //退換貨單價
        { name: 'FRWH', type: 'string' }
    ]
});